src/test/scala/cornichonTests/CheckoutTest.scala

```diff
@@ -3,8 +3,14 @@ import com.github.agourlay.cornichon.core.FeatureDef
 
 class CheckoutTest extends FeatureWithToken {
 
+  override lazy val baseUrl = apiUrl + s"/$projectKey"
+
   def feature: FeatureDef = Feature("Checkout process") {
-    Scenario("Adding item to cart | Making order from cart | Deleting order, cart, tax") {
+    Scenario(
+      """Create cart | Create tax category | Add line item |
+        | Create order | Delete order | Delete cart |
+        | Delete tax category
+        | """.stripMargin.replaceAll("\n","")) {
```

Comment:
Maybe we can find a more functional name, like `ordering a product external to the platform`

Reply:
Do you mean the feature name or the scenario? I thought to describe the main steps in Scenario, it's a little verbose though, maybe something like - "Create cart | Add custom line item | Create order |  Cleanup" ?

Reply:
yes it's a bit verbose.
I like your proposal.

Reply:
@yanns